docs/en/inference/vl_pipeline.md

```diff
@@ -148,6 +148,24 @@ response = pipe(('describe this image', image))
 print(response)
 ```
 
+### Calculate logits
+
+We support custom inputs, users can use `prepare_inputs` to get a understand how the inputs is organized.
```

Comment:
gpt: 
We provide support for custom inputs. Users can utilize 'prepare_inputs' to understand how the inputs are organized.